src/components/KDateTimePicker/KDateTimePicker.vue

```diff
@@ -804,33 +811,39 @@ $margin: 6px;
       .vc-weeks {
         margin-top: color(spacing-sm);
         .vc-weekday {
-          color: $text-color;
+          color: $text-color-darker;
         }
       }
     }
     .vc-pane-container,
     .vc-time-picker {
       // Time Range
       .vc-select select {
+        background-color: color(grey-200);
         color: $text-color-darker;
         &:hover {
           color: $text-color-darker;
         }
         &:focus {
           color: $text-color-darker;
-          background-color: color(gray-200);
-          border: 2px solid var(--white);
+          background-color: color(grey-200);
+          border: 2px solid color(grey-200);
+
+          + .vc-select-arrow {
+            color: color(grey-500);
+          }
         }
       }
       .vc-time-month, .vc-time-day, .vc-time-year {
-        color: $text-color;
+        color: $text-color-darker;
       }
       .vc-month, .vc-day {
-        color: $text-color;
+        color: $text-color-darker;
       }
 
       // AM / PM highlights
       .vc-am-pm {
+        background-color: color(grey-200);
```

Comment:
Since we're using `color(grey-200)` in four instances, can you please create a `$text-color-lighter: color(grey-200);` variable above `$text-color: color(grey-500);` and use that variable throughout?

It's slightly cleaner, and helps us push back if UX wants yet another shade of gray in the future. We can just say "we have 3 to choose from, pick one" ;)

Reply:
Good idea @mihai-peteu! Added those changes, please take a look :)